src/store/Store.js

```diff
@@ -10,12 +10,12 @@ class Store {
    * Creates a new Proxy store
    * @param  {object} options An object of form {portName, state}, where `portName` is a required string and defines the name of the port for state transition changes and `state` is the initial state of this store (default `{}`)
    */
-  constructor({portName, state = {}}) {
+  constructor({portName, state = {}, extensionId = ''}) {
     if (!portName) {
       throw new Error('portName is required in options');
     }
-
-    this.port = chrome.runtime.connect({name: portName});
+    this.extensionId = extensionId; //keep the extensionId as an instance variable
+    this.port = chrome.runtime.connect(this.extensionId, {name: portName});
```

Comment:
I have yes, I tried it with your original `clicker-key` example happening in the `content.js`.